plugin.yml

```diff
@@ -14,6 +14,7 @@ configuration:
     push:
       type: [ string, array ]
       minimum: 1
+
```

Comment:
Checking this additional line is expected to break up the plugin "commands" from the additional arguements?

Reply:
I added it on purpose to separate the _"main commands"_ from the rest of the options that are sorted alphabetically